Backend/api/rest_views/classroom.py

```diff
@@ -0,0 +1,10 @@
+
+from grest import GRest
+from models import ClassRoom
+
+
+class ClassRoomRestView(GRest):
+    """Person's View (/rest/classroom)"""
+    __model__ = {"primary": ClassRoom}
+    #__selection_field__ = {"primary": "name"} ca n-are fields
```

Comment:
classroom inherits waypoint deci are toate field-urile din waypoint

Reply:
in regula, asa si la connector, acolo m-am prins de mostenire